### PR TITLE
Fix #144680 ylilauta.org

### LINKED ISSUE #144680: ylilauta.org
### Issue URL (Incorrect Blocking)
 [https://ylilauta.org/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fylilauta.org%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Safari v1.11.10
Browser: | Safari
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base, <br/>AdGuard Mobile Ads, <br/>EasyList<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection, <br/>AdGuard URL Tracking<br/><br/><b>Language-specific:</b><br/>Adblock List for Finland

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/144680

```javascript
(function() {
    var b = window.alert;
    window.alert = function(a) {
        if (!/you are using an ad blocker/.test(a.toString())) return b(a)
    };
})();
```

### CODE REVIEW

**Review by piquark6046 — COMMENTED** [review]
I think that changing `var` statement to `const` declaration should be performed for safety because `const` declaration makes the value a constant.
 - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const
 - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/var

**Review by zloyden — COMMENTED** [review]
Btw, how to reproduce it?

**Comment by jellizaveta** [comment]
Played with the settings from this comment https://github.com/AdguardTeam/AdguardFilters/issues/144680#issuecomment-1457249243

> Btw, how to reproduce it?

Reproduced with the settings from this comment https://github.com/AdguardTeam/AdguardFilters/issues/144680#issuecomment-1457249243

**Comment by jellizaveta** [comment]
> I think that changing `var` statement to `const` declaration for safety because `const` declaration makes the value a constant.
> 
> * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const
> * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/var

agree , should be declared with const

**Review by piquark6046 — APPROVED** [review]

File: BaseFilter/sections/foreign.txt [hunk 1]
```diff
@@ -2019,6 +2019,8 @@ ylilauta.org#%#//scriptlet('abort-on-stack-trace', 'Element.prototype.append', '
 ||static.ylilauta.org/js/*/Module/FrameLoader.js$replace=/this\.#?(contentLoadFailed|showFail)\(\);//
 ||ylilauta.org^$document,replace=/(<script src="https:\/\/static\.ylilauta\.org\/js\/.+?\/Module\/FrameLoader\.js" )integrity=".*?" (type="module"><\/script>)/\$1\$2/
 ||ylilauta.org^$removeheader=link
+ylilauta.org#%#(function(){const b=window.alert;window.alert=function(a){if(!/you are using an ad blocker/.test(a.toString()))return b(a)};})();
+ylilauta.org##.toast-root
```
Comment (zloyden) [inline BaseFilter/sections/foreign.txt:2023]: Have you checked these rules with the app?
Seems they are not needed on my end.
I'd limit them only for extensions.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: BaseFilter/sections/foreign.txt (hunk 1 above)
Comment (zloyden) [reply BaseFilter/sections/foreign.txt:2024]: I meant this directive

```
!#if (!adguard_app_windows && !adguard_app_mac && !adguard_app_android)
```

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**